html-aam/index.html

```diff
@@ -8680,7 +8715,7 @@ <h4 id="att-autofocus">`autofocus`</h4>
             </tr>
             <tr>
               <th>Comments</th>
-              <td>Provides a <a href="termref">minimum role</a> of <a class="core-mapping" href="#role-map-group">`group`</a>.</td>
+              <td>Provides a <a>minimum role</a> of <a class="core-mapping" href="#role-map-group">`group`</a>.</td>
```

Comment:
Did you intend to remove href="termref" here and in the other place?

Reply:
yes.  it was broken, this fixed the link